### PR TITLE
w-11639754

### PR DESCRIPTION
**Pull request by PabloCamino**
Adding this troubleshooting entry after W-11179827

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by IsaacEldridge — COMMENTED** [review]
@PabloCamino Sorry I missed this. Can you please take a look at my edits. Thanks

**Comment by PabloCamino** [comment]
Looks great to me!

File: modules/ROOT/pages/troubleshoot-guide.adoc [hunk 1]
```diff
@@ -447,3 +447,29 @@ Scenario: After an appliance upgrade, you lose customizations made to your email
 
 This is a limitation in Runtime Fabric on VMs / Bare Metal, as documented in xref:upgrade-index.adoc#before-you-upgrade-the-runtime-fabric-appliance[before you upgrade the Runtime Fabric appliance]. The suggested workaround is to back up any customizations before you run the upgrade and then reapply the customizations when the upgrade completes.
 
+== Troubleshoot External Log Forwarding
+
+=== Log Aggregators Must Use TLS v1.2 
+
+Runtime Fabric external log forwarding libraries does not support TLS v1.3.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/troubleshoot-guide.adoc]:
```suggestion
Runtime Fabric external log forwarding libraries do not support TLS v1.3.
```

**Review by sathya0 — APPROVED** [review]

**Comment by IsaacEldridge** [comment]
sorry @PabloCamino I forgot to merge this.

**Pull request merged**